README.md

```diff
@@ -1,54 +1,31 @@
 # Arbiter (In Progress)
 
 ![](https://visitor-badge.laobi.icu/badge?page_id=arbiter)
-![Github Actions](https://github.com/primitivefinance/arbiter/workflows/Rust/badge.svg) 
+![Github Actions](https://github.com/primitivefinance/arbiter/workflows/Rust/badge.svg)
 [![](https://dcbadge.vercel.app/api/server/primitive?style=flat)](https://discord.gg/primitive)
 [![Twitter Badge](https://badgen.net/badge/icon/twitter?icon=twitter&label)](https://twitter.com/primitivefi)
 
-> Code that runs code that runs code to study how code runs code and behaves in any environment.
+> Perform high speed modeling and economic fuzzing with EVM parity.
 
-Ethereum's execution environment, the Ethereum virtual machine (EVM), has given fruit to a rich collection of decentralized applications. The EVM resembles a simple stack machine and sequentially executes opcodes as decentralized applications are used, deployed, or exploited.
+Ethereum's execution environment, the Ethereum virtual machine (EVM), has given fruit to a rich collection of decentralized applications. The EVM resembles a simple stack machine and sequentially executes opcodes as decentralized applications are used, deployed, or exploited. Arbiter is a highly configurable rust interface over [REVM](https://github.com/bluealloy/revm).
```

Comment:
Everywhere else we just say "revm" and not REVM. The repo itself uses lowercase.